torch/ao/quantization/fx/utils.py

```diff
@@ -247,7 +260,7 @@ def quantize_node(
         module_path = ""
     root_module = modules['']
     graph = quantized_graph
-    maybe_quantize_node_info = get_quantize_node_info(obs_module)
+    maybe_quantize_node_info = get_quantize_node_info(obs_module, False)
```

Comment:
can we do something like

```
is_decomposed_qtensor = False
maybe_quantize_node_info = get_quantize_node_info(obs_module, False)
```

to make it easy to understand what this flag is without having to go into the function definition

Reply:
oh this code is going to be removed in next PR :)

Reply:
updated this one as well, since it's a small change anyways..